custom-connectors/ServiceNow/apiDefinition.swagger.json

```diff
@@ -0,0 +1,3130 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "ServiceNow Custom Sample",
+    "description": "A sample connector for ServiceNow with basic CRUD for Incidents, OAuth and attachment support",
+    "contact":{
+      "name": "https://www.microsoft.com/",
+      "url": "https://support.microsoft.com/",
```

Comment:
I dont think Microsoft Support will handle this. Lets remove this.

Reply:
have removed Microsoft contact info.